### PR TITLE
allow sending view once ptt audio messages

### PR DESCRIPTION
**Pull request by RustyStriker**
seems simply passing forward the isViewOnce in chat.sendFileMessage works for ptt audio messages

### CODE REVIEW

**Comment by icleitoncosta** [comment]
Hello, congratulations on your solution and the commit.
Thank you!

It gave some lint errors, please declare the variable in the AudioMessageOptions interface, in the same file.

**Comment by RustyStriker** [comment]
@icleitoncosta Done, anything else?

File: src/chat/functions/sendFileMessage.ts [hunk 1]
```diff
@@ -265,7 +266,11 @@ export async function sendFileMessage(
   let isViewOnce: boolean | undefined;
 
   if (options.type === 'audio') {
+    if(options.isPtt) {
```
Comment (icleitoncosta) [inline src/chat/functions/sendFileMessage.ts:269]: Add space between if and (.

File: src/chat/functions/sendFileMessage.ts [hunk 2]
```diff
@@ -265,7 +266,11 @@ export async function sendFileMessage(
...
+      isViewOnce = options.isViewOnce;
+    }
     rawMediaOptions.isPtt = options.isPtt;
+
```
Comment (icleitoncosta) [inline src/chat/functions/sendFileMessage.ts:273]: Delete spaces from this lines

**Review by icleitoncosta — CHANGES_REQUESTED** [review]

**Pull request closed without merging**